jsonnet/kube-prometheus/addons/windows-hostprocess.libsonnet

```diff
@@ -0,0 +1,197 @@
+local windowsdashboards = import 'github.com/kubernetes-monitoring/kubernetes-mixin/dashboards/windows.libsonnet';
+local windowsrules = import 'github.com/kubernetes-monitoring/kubernetes-mixin/rules/windows.libsonnet';
+
+local defaults = {
+  local defaults = self,
+  // Convention: Top-level fields related to CRDs are public, other fields are hidden
+  // If there is no CRD for the component, everything is hidden in defaults.
+  name:: 'windows-exporter',
+  namespace:: error 'must provide namespace',
+  version:: error 'must provide version',
+  image:: error 'must provide version',
+  resources:: {
+    requests: { cpu: '300m', memory: '200Mi' },
+    limits: { memory: '200Mi' },
```

Comment:
how did you choose these vaules?  They might be a bit low for Windows?

Reply:
I'm running it from my fork in few windows clusters.
I see the usage as below. The scrape duration that I'm getting for windows exporters is between 400-700ms.

NAME                     CPU(cores)   MEMORY(bytes)
windows-exporter-2lw7j   24m          31Mi
windows-exporter-8vxc9   22m          31Mi
windows-exporter-bbpc8   17m          31Mi
windows-exporter-c7qgw   22m          31Mi
windows-exporter-gmq4p   15m          29Mi
windows-exporter-r4zbk   21m          30Mi
windows-exporter-r7kms   17m          29Mi
windows-exporter-rhrfn   19m          30Mi
windows-exporter-t2r9l   21m          29Mi
windows-exporter-tb699   19m          32Mi
windows-exporter-tmbz2   19m          30Mi
windows-exporter-xk8n6   21m          32Mi

Reply:
Is that idle or with pods running on the system?  Just want to make sure we aren't too low out of the box, I've seen other windows "system" pods hit OOM limits when under load.  Based on what you shared, this looks reasonable to start with.

Reply:
This is a non-prod cluster but it has 60 pods per e8s node. I just deployed it in our prod cluster which is running 35 pods per node but has higher CPU utilization. And I see similar numbers. I'm guessing since it is a host process pod it does not spin up other system services per pod (csrss,svchost etc). So the utilization is just for windows_exporter process.

Reply:
> I'm guessing since it is a host process pod it does not spin up other system services per pod (csrss,svchost etc). So the utilization is just for windows_exporter process.

This makes sense.  Thanks for verifying!